src/main/scala/gitbucket/core/service/RepositoryService.scala

```diff
@@ -416,6 +423,30 @@ trait RepositoryService { self: AccountService =>
     }
     .sortBy(_.userName asc).map(t => t.userName -> t.repositoryName).list
 
+  /**
+    * Returns content of template set per repository.
+    *
+    * @param repository the repository information
+    * @param fileBaseName the file basename of template
+    * @return The content of template if the repository has it, otherwise empty string.
+   */
+  def getContentTemplate(repository: RepositoryInfo, fileBaseName: String)(implicit s: Session): String = {
+    val preferred = PluginRegistry().renderableExtensions.map(extension => s"${fileBaseName}.${extension}")
+
+    def choiceTemplate(files: List[FileInfo]): Option[FileInfo] =
+      files.find(f => preferred.contains(f.name)).orElse(files.find(f => f.name.startsWith(fileBaseName)))
```

Comment:
why not filtering with exact naming (with or without extension)?
here I assume we can retrieve file names like : `ISSUE` or ISSUE_TEMPLATE_THAT_SHOULD_NOT_BE_SELECTED